transforms/code/repo_level_ordering/ray/README.md

```diff
@@ -0,0 +1,159 @@
+* `run-local-sample` - runs src/repo_level_order_local_ray.py
+
+These targets will activate the virtual environment and set up any configuration needed.
+Use the `-n` option of `make` to see the detail of what is done to run the sample.
+
+For example, 
+```shell
+make run-cli-sample
+...
+```
+Then 
+```shell
+ls output
+```
+To see results of the transform.
+
+
+## Using the CLI
+
+Simplest Usage:
+
+All data in input is divided by parquet 
+
+```sh
+export INPUT_FOLDER="input_data_folder/"
+export OUTPUT_FOLDER="output_data_folder/"
+
+
+local_conf="{'input_folder' : '$INPUT_FOLDER', 'output_folder' : '$OUTPUT_FOLDER'  }"
+
+python src/repo_level_order_transform_ray.py \
+       --run_locally True \
+       --data_local_config "$local_conf" \
+       --repo_lvl_store_type  local  \
+       --repo_lvl_store_backend_dir '/tmp/mystore' \
+       --repo_lvl_sorting_enabled \
+       --repo_lvl_sorting_algo SORT_SEMANTIC \
+```
+
+
+Case 1:
+
+When we want the output with the following configuration enabled:
+- repos should be sorted by semantic order
+- output should be grouped by dominant language per repo
+- all files of repo must be combined to a single row in the output.
+- data is local
+- store backend : local filsystem
+(store_backend: s3/local are persistent stores and retain the mappings stores in them, they need to be cleaned/deleted after use. It is recommented to use a different location for store if data is different. They are added to aid in large data processing in multiple stages.
+
+store_backend: ray, is not presistent and ephimeral and does not retain any data.
+  It is the simplest to use if resources are available.
+)
+
+```sh
+export INPUT_FOLDER="input_data_folder/"
+export OUTPUT_FOLDER="output_data_folder/"
+
+
+local_conf="{'input_folder' : '$INPUT_FOLDER', 'output_folder' : '$OUTPUT_FOLDER'  }"
+
+python src/repo_level_order_transform_ray.py \
+       --run_locally True \
+       --data_local_config "$local_conf" \
+       --repo_lvl_store_type  local  \
+       --repo_lvl_store_backend_dir '/tmp/mystore' \
+       --repo_lvl_combine_rows \
+       --repo_lvl_sorting_enabled \
+       --repo_lvl_sorting_algo SORT_SEMANTIC \
+       --repo_lvl_output_by_langs
+        
+```
+
+Case 2:
+
+When data is on cloud and 
+
+```shell
+
+
+export INPUT_FOLDER="input_bucket/input_dir"
+export OUTPUT_FOLDER="output_bucket/output_dir"
+
+s3_kreds="{ ... }" # in the deafult way used for all transforms. 
+s3_conf="{'input_folder' : '$INPUT_FOLDER', 'output_folder' : '$OUTPUT_FOLDER'  }"
+
+python src/repo_level_order_transform_ray.py \
+       --run_locally True \
+       --data_s3_cred "$s3_kreds" \
+       --data_s3_config "$s3_conf" \
+       --repo_lvl_store_type  local  \
+       --repo_lvl_store_backend_dir '/tmp/mystore' \
+       --repo_lvl_combine_rows \
+       --repo_lvl_sorting_enabled \
+       --repo_lvl_sorting_algo SORT_SEMANTIC \
+       --repo_lvl_output_by_langs   
+```
```

Comment:
You should also add the new section on running transform images...see https://github.com/IBM/data-prep-kit/tree/dev/transforms/code/code2parquet/python#to-see-results-of-the-transform